gramps/plugins/lib/libgedcom.py

```diff
@@ -1129,14 +1129,12 @@ def __init__(self):
 
         try:
             filepath = os.path.join(DATA_DIR, "gedcom.xml")
-            ged_file = open(filepath, "rb")
+            with open(filepath, "rb") as ged_file:
+                parser = GedInfoParser(self)
```

Comment:
Since it was originally around just the `open`, it's probably checking for non-existent files. Maybe simpler to do `if not os.path.exists(filepath): return`.